integ-test/src/test/java/org/opensearch/sql/sql/PaginationIT.java

```diff
@@ -76,4 +81,32 @@ public void testCursorTimeout() throws IOException, InterruptedException {
 
     wipeAllClusterSettings();
   }
+
+  @Test
+  @SneakyThrows
+  public void testCloseCursor() {
+    var query = "SELECT * from " + TEST_INDEX_CALCS;
+    var response = new JSONObject(executeFetchQuery(query, 4, "jdbc"));
+    assertTrue(response.has("cursor"));
+    var cursor = response.getString("cursor");
+
+    Request closeCursorRequest = new Request("POST", "_plugins/_sql/close");
```

Comment:
api resource should be `_close`?  Or is this following legacy API?

Reply:
Not a mistake:
https://github.com/opensearch-project/sql/blob/8e5d766b0fbea11b449eeada9fb1ccab375cec1f/legacy/src/main/java/org/opensearch/sql/legacy/plugin/RestSqlAction.java#L78-L80